uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -734,6 +734,30 @@ public function upgrade_1032() {
     return TRUE;
   }
 
+  /**
+   * Update permissions and set the weight for the "Tasks" menu item.
+   *
+   * @return bool
+   */
+  public function upgrade_1033() {
+    $params = ['return' => 'id', 'name' => 'tasksassignments_administer'];
+    $parentId = (int) civicrm_api3('Navigation', 'getvalue', $params);
+    $params = ['return' => 'id', 'name' => 'ta_settings'];
+    $taSettingsId = (int) civicrm_api3('Navigation', 'getvalue', $params);
```

Comment:
Why don't we just call the Navigation API once to get the ids of the menu's rather than twice?